src/libs/actions/Report.js

```diff
@@ -524,6 +524,26 @@ function openPaymentDetailsPage(chatReportID, iouReportID) {
     });
 }
 
+/**
+ * Gets metadata info about links in provided report
+ *
+ * @param {String} reportID
+ * @param {String} reportActionID
+ */
+function expandURLPreview(reportID, reportActionID) {
+    API.read('ExpandURLPreview', {
+        reportID,
+        reportActionID,
+    }, {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportActionID}`,
+            },
```

Comment:
This usually has a `value` field. e.g. if you want to have some loading state happening we could do this here or if we knew the shape of the data we expect to receive we could put that here. But in this case maybe the entire `optimisticData` can just be removed.